app/src/main/java/com/nilhcem/blenamebadge/device/bluetooth/ScanHelper.kt

```diff
@@ -54,7 +55,12 @@ class ScanHelper {
                 .setScanMode(ScanSettings.SCAN_MODE_LOW_LATENCY)
                 .build()
 
-        scanner.startScan(filters, settings, scanCallback)
+        //check if bluetooth is enabled
+        if (BluetoothAdapter.getDefaultAdapter().isEnabled) {
+            scanner.startScan(filters, settings, scanCallback)
+        } else {
+            Timber.i { "Turn on Bluetooth and try again" }
```

Comment:
There's no information to user that what he/she should do!

Show a toast

Reply:
Done!